cliquet/resource.py

```diff
@@ -282,7 +290,8 @@ def collection_delete(self):
         filters = self._extract_filters()
         deleted = self.collection.delete_records(filters=filters)
 
-        return self.postprocess(deleted)
+        action = len(deleted) > 0 and ACTION_DELETE or ACTION_READ
```

Comment:
Do we really want to trigger the even only if something was deleted? I think this should be up to the even handler to decide with the related data what to do in that case.

Anyway why would you like to send a read action in that case?